custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -194,6 +194,31 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
         vm.selectedLocations = new Array(maxLevel);
     };
 
+    vm.userHaveAccessToAllLocations = function (locations) {
+        var haveAccessToAllLocationsForLevel = true;
+        window.angular.forEach(locations, function (location) {
```

Comment:
Why is this using `window.angular.forEach` when other parts of this PR use underscore iterators? I think this would be equivalent and simpler if the whole function body were replaced with `return _.find(locations, function (location) { return !location.user_have_access; });`